openedx/openedx-platform
# feat: New link on studio home to Taxonomy list and flag added [FC-0036]

## Description

Added a new waffle flag to enable a new link on Studio Home to redirect to the Taxonomy List page on the Course authoring MFE.

## Supporting information

- Internal ticket: [FAL-3522](https://tasks.opencraft.com/browse/FAL-3522)
- frontend-app-course-authoring realted PR: https://github.com/openedx/frontend-app-course-authoring/pull/622
- openedx-learning realted PR: https://github.com/openedx/openedx-learning/pull/98
- Issue ticket: https://github.com/openedx/modular-learning/issues/104

## Testing instructions

**Setup**

- Set `ENABLE_TAGGING_TAXONOMY_PAGES`  for the frontend-app-course-authoring by adding ENABLE_TAGGING_TAXONOMY_PAGES=true to devstack/microfrontend.yml.
-  Turn on the `new_studio_mfe.use_tagging_taxonomy_list_page` flag on [studio](http://localhost:18000/admin/waffle/flag/).
- Use this branch of frontend-app-course-authoring: https://github.com/openedx/frontend-app-course-authoring/pull/622
- Follow [setup steps for frontend-lib-content-components](https://github.com/openedx/frontend-lib-content-components#how-to-set-up-development-workflow-of-v2-content-editors-in-studio-and-course-authoring-mfe). You can skip `1.I` and `1.III`
- On CMS shell run `make requirements` and restart the CMS server.

**Testing**

- Go to [Studio Home](http://localhost:18010/home/) with staff user.
- Click on the `Taxonomies` link next to `Libraries` link

![image](https://github.com/openedx/edx-platform/assets/11827305/f3ab145c-f59e-444c-8d77-16dfca98e38a)

- Verify that the page is redirected to the Taxonomy list page
- Continue reading the testing instructions on https://github.com/openedx/frontend-app-course-authoring/pull/622


## Other information

**Before merge**

- [x] Merge https://github.com/openedx/openedx-learning/pull/98
- [x] Bump openedx-learning
